scripts/kind-e2e/e2e.sh

```diff
@@ -37,7 +37,7 @@ with_context cluster3 subctl_install_subm
 
 verify="--connectivity"
 if [[ "$lighthouse" == "true" ]]; then
-    verify="--all"
+    verify="--service-discovery"
```

Comment:
This will still run only SD tests, or am I missing something?

Maybe you meant
```suggestion
    verify+=" --service-discovery"
```

Reply:
> This will still run only SD tests, or am I missing something?
> 
> Maybe you meant

conenctivity flag is set to true by default. This is setting service-discovery flag to true. with both true, it is basically all.

Reply:
OK just with line 38 it's a bit confusing, maybe that line should be removed

Reply:
I'd use mike's suggestion, it's at least more clear :) although we will be changing it soon